### PR TITLE
Routing, Homepage, and Nav bar with login

### PR DESCRIPTION
**Pull request by Rhaypapenfuzz**

### CODE REVIEW

File: src/Routing.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  function Profile() {
+    return (
+      <div>
+        {MediaCard()}
+      </div>
+    );
+  }
+  
+  function Game(){
+    const dict = {userId : "kricheso@google.com", gridId : "board1_by_kricheso@google.com"}
```
Comment (kricheso) [inline src/Routing.js:32]: Next step: remove all hardcoded values

File: src/nav.js [hunk 2]
```diff
@@ -87,6 +91,15 @@ export default function PersistentDrawerLeft() {
     setOpen(false);
   };
 
+  const [user, setUser] = useState(null);
+
+  async function loginPress(){
+    const user = await Authentication.logIn();
+    if (user === null) { /* error */ }
```
Comment (kricheso) [inline src/nav.js:98]: Can you display to the user that the log in failed? Using an alert would be a minimum. A custom UI component or text indicating so would be ideal.

**Review by kricheso — APPROVED** [review]
Approved. Please fix issues in future commits.

File: src/Routing.js (hunk 1 above)
Comment (brybusti) [reply src/Routing.js:32]: I can't call play if I don't have that though.

Comment (kricheso) [reply src/Routing.js:32]: Try const dict = {userId : user.id, gridId : grid.id}. See example on how to do this in firestoremanualtests.js file. (You will need to figure out the grid part).

**Pull request merged**